modules/affile/affile-source.c

```diff
@@ -253,6 +253,43 @@ affile_sd_construct_proto(AFFileSourceDriver *self, gint fd)
     }
 }
 
+static void
+affile_sd_reset_file_state(PersistState *state, const gchar *old_persist_name)
+{
+  gchar *new_persist_name = g_strdup_printf("%s_DELETED",old_persist_name);
+  persist_state_rename_entry(state,old_persist_name,new_persist_name);
+  g_free(new_persist_name);
+}
+
+static inline gboolean
+_setting_logpipe(LogPipe *s)
```

Comment:
also, the name of the function doesn't tell  me anything and it should. it sets up the LogReader to send its output to our LogPipe instance. 

also since this is not a public API function, the type of the argument could use specific type (AFFileSourceDriver instead of LogPipe).